modules/log_forwarder/main.tf

```diff
@@ -96,7 +96,7 @@ resource "aws_iam_policy" "this" {
     {
       vpc_check             = var.subnet_ids != null
       s3_check              = length(var.s3_log_bucket_arns) > 0
-      s3_log_bucket_arns    = jsonencode(var.s3_log_bucket_arns)
+      s3_log_bucket_arns    = jsonencode(concat(var.s3_log_bucket_arns, formatlist("%s/*", var.s3_log_bucket_arns)))
```

Comment:
we don't want to manipulate the ARNs like this since its not obvious to users. its up to users to pass the appropriate ARNs, including any path prefixes or wildcards

Reply:
That makes sense, thanks @bryantbiggs